deployment/constants.py

```diff
@@ -63,4 +64,30 @@
     TAPIR: "https://porter-tapir.nucypher.io/get_ursulas",
 }
 
+#
+# Domain Seednodes
+#
+
+NETWORK_SEEDNODE = {
```

Comment:
Just to keep the name clear (since we have used seed node as a CLI parameter):
```suggestion
NETWORK_SEEDNODE_STATUS_JSON = {
```

or something else a bit more descriptive.